### PR TITLE
Add python solution to cses: Two Sets II

### PR DESCRIPTION
**Pull request by Amaryllidaceae**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [X] I have tested my code.
- [X] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [X] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

firefox_Q2m2I8cTad

### CODE REVIEW

File: solutions/cses-1093.mdx [hunk 1]
```diff
@@ -127,4 +127,44 @@ public class TwoSets {
...
 ```
 </JavaSection>
+
+<PySection>
+
+```py
+MOD = 10 ** 9 + 7
+
+
+def solve(n: int) -> int:
```
Comment (SansPapyrus683) [inline solutions/cses-1093.mdx:137]: why in function

**Review by dongliu0426 — APPROVED** [review]

File: solutions/cses-1093.mdx (hunk 1 above)
Comment (Amaryllidaceae) [reply solutions/cses-1093.mdx:137]: It's easy to break the input into two cases, odd and even, and return them at different points in the code. Although it's just as easy to remove the function I suppose. Do you want me to do that?

Comment (SansPapyrus683) [reply solutions/cses-1093.mdx:137]: yeah, do that & `sys.exit()`

File: solutions/cses-1093.mdx [hunk 2]
```diff
@@ -127,4 +127,42 @@ public class TwoSets {
...
+import sys
+
+MOD = 10 ** 9 + 7
+
+n = int(input())
+
+sum_elem = n * (n + 1) // 2  # Sum of elements in [1, ..., n]
+
+# If the sum of the elements is odd, then we clearly can't split it evenly.
+if sum_elem & 1:
```
Comment (SansPapyrus683) [inline solutions/cses-1093.mdx:143]:
```suggestion
if sum_elem % 2 == 0:
```
why the bitwise?

Comment (dongliu0426) [reply solutions/cses-1093.mdx:143]:
```suggestion
if sum_elem % 2 == 1:
```

**Pull request merged**